<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="menuWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>60</y>
      <width>800</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="w_main">
     <widget class="QTextBrowser" name="title">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>801</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0%);
border: 0px;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;GameCeption&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;A game within a game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_singleplayer">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Single Player</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_multiplayer">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>240</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Multi Player</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_about">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>340</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>About</string>
      </property>
     </widget>
     <widget class="QLabel" name="JackpotLabel_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>120</y>
        <width>131</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Images/Images/Cricket.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="JackpotLabel_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Images/Images/android_quiz.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="JackpotLabel_6">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>-110</y>
        <width>131</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Images/Images/Stone.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="JackpotLabel_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>-100</y>
        <width>131</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Images/Images/jackpot.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="JackpotLabel_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Images/Images/whack-a-mole2-300x201.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="w_play">
     <widget class="QPushButton" name="b_end_game">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>500</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>End Game</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>591</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="g_board" columnminimumwidth="0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="t_result">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0%);
border: 0px;
font: 24pt &quot;Sans Serif&quot;;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:24pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
